install/operator/pkg/syndesis/configuration/configuration_test.go

```diff
@@ -88,6 +88,15 @@ func TestSpecificConfig(t *testing.T) {
 						VolumeCapacity: "7Gi",
 					},
 				},
+				Grafana: v1alpha1.GrafanaConfiguration{
+					Resources: v1alpha1.Resources{
+						ResourceRequirements: v12.ResourceRequirements{
+							Limits: v12.ResourceList{
+								"memory": resource.MustParse("2Gi"),
```

Comment:
Isn't this around 2Gb?? I would suggest something closer to the 256Mb.

Reply:
Yeah, it's just a test case so I picked a random value. I can modify it to 256Mb though.

I actually created a new template param `GRAFANA_MEMORY_LIMIT` which defaults to 512Mb which I can also modify to 256Mb if you think it's better.

Reply:
Oh I see, 512Mb is more reasonable dough ... great stuff

Reply:
512Mb makes more sense, great stuff!